lesson_7/mihaela_vaida/print_register_dec.py

```diff
@@ -0,0 +1,21 @@
+
+print_registry = []
+def register(fnc):
+    
+    
+    def print_register(*args):        
+        print_registry.append(fnc.__name__)
+        print(print_registry)
+    return print_register   
```

Comment:
This does what was requested, it adds the function to the registry, but it also does something extra ... 
Can you still use the decorated functions as they were intended?

Reply:
I do not understand  what  do you mean by   something else is  extra done in this exercise , because I tried to check  it but the result was   :
 greet('dsdas')  # ['greet']
say_hello('Vio')  # 'Hello Vio!'- this  is the only  un-decorated  function
say_goodbye('sads')  # ['greet', 'say_goodbye']
greet('dsdas') 
['greet', 'say_goodbye', 'greet']
greet('dsdas') 
['greet', 'say_goodbye', 'greet', 'greet']
say_goodbye('sads')
['greet', 'say_goodbye', 'greet', 'greet', 'say_goodbye']